src/main/java/duke/Event.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.time.LocalDate;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Class representing events.
+ */
+class Event extends Task {
+
+    /**
+     * Variable to store event time.
+     */
+    protected LocalDate at;
```

Comment:
Perhaps can consider using a more descriptive name? I notice the same issue in the Deadline class.